.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -14,25 +14,21 @@ assignees: amarpad, ekowtaylor, talkhasib
 - **Affected Subcomponent:** Orchestrator pod/service or AGW service
 - **Deployment Environment:** e.g. AWS (orc8r), local docker-compose (orc8r), bare-metal (AGW), Vagrant (AGW)
 
-### Describe the Feature
+### Describe the problem
 
-**Is your feature request related to a known issue? Please describe.**
-A clear and concise description of what the problem is and provide links to any relevant issues
+**Description**
+A clear and concise description of what the problem is, under which scenarios or how frequent it happen
 
-**Specify the applicable product use-cases**
-Magma product use-case support is growing, specify what use-cases apply including FWA, CWF, PLTE, MBB etc.
+**Additional context**
+Add any other context or screenshots about the problem here. You can also provide links to relevant resources.
 
-**Describe the solution you'd like**
-A clear and concise description of your preferred solution including links to design docs/proposals will be great.
+### Describe your idea(optional)
```

Comment:
@wallyrb missing space after "idea" and before the paren 🙂

Reply:
I missed that, thanks @hcgatewood! Here the PR https://github.com/magma/magma/pull/8968 for fixing that